### PR TITLE
Update Known-issues.md

### PR DESCRIPTION
**Pull request by dariomws**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/4117

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/Known-issues.md [hunk 1]
```diff
@@ -257,7 +257,7 @@ This article lists the known issues for Microsoft Teams, by feature area.
...
+|Resource Account misconfigured Department <br/> | On premise resource Accounts associated with an auto attendant or call queue created before January 2019 might not have the Department parameter set properly, which might cause a phone number assignment to fail. A fix is underway to resolve this issue. <br/><br/> Resource Accounts configured using New-CsHybridApplicationEndpoint with Skype for Business Server may not have the Department parameter set properly which will cause the resource account creation in Teams admin center to fail. In this case, you need to configure the department name in Active Directory on premise before synchronizing to online.|To mitigate this issue, you can run the following Cmdlet to set the department parameter: Set-AdUser -Identity <objectId> -Department “Microsoft Communication Application Instance” <br/> Also see the [Auto Attendant and Call Queues Service Update](https://techcommunity.microsoft.com/t5/Microsoft-Teams-Blog/Auto-Attendant-and-Call-Queues-Service-Update/ba-p/564521). |5/8/19 <br/> |
```
Comment (JohanFreelancer9) [inline Teams/Known-issues.md]:
```suggestion
|Resource account misconfigured Department <br/> | On-premises resource accounts associated with an auto attendant or call queue created before January 2019 might not have the Department parameter set properly, which might cause a phone number assignment to fail. A fix is underway to resolve this issue. <br/><br/> Resource Accounts configured using New-CsHybridApplicationEndpoint with Skype for Business Server may not have the Department parameter set properly which will cause the resource account creation in the Teams admin center to fail. In this case, you need to configure the department name in Active Directory on-premises before synchronizing to online.|To mitigate this issue, you can run the following Cmdlet to set the Department parameter: Set-AdUser -Identity <objectId> -Department “Microsoft Communication Application Instance” <br/> Also see the [Auto Attendant and Call Queues Service Update](https://techcommunity.microsoft.com/t5/Microsoft-Teams-Blog/Auto-Attendant-and-Call-Queues-Service-Update/ba-p/564521). |5/8/19 <br/> |
```

**Comment by scanum** [comment]
@LolaJacobsen PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LolaJacobsen** [comment]
@marclauinger please review.

**Pull request merged**